shared/fs/reset-banner/container.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as Constants from '../../constants/fs'
+import * as FsGen from '../../actions/fs-gen'
+import * as RPCTypes from '../../constants/types/rpc-gen'
+import {compose, connect, setDisplayName, type TypedState, type Dispatch} from '../../util/container'
+import {isMobile} from '../../constants/platform'
+import {createShowUserProfile} from '../../actions/profile-gen'
+import {createGetProfile} from '../../actions/tracker-gen.js'
+import Banner from '.'
+
+const mapStateToProps = (state: TypedState, {path}) => {
+  const _pathItem = state.fs.pathItems.get(path, Constants.makeUnknownPathItem())
+  const _username = state.config.username || undefined
+  return {
+    _pathItem,
+    _username,
+    path,
+    isUserReset:
```

Comment:
if you're going to bother to pass down _pathitem and _username you can calculate this in mergeProps instead